frontend/templates/download.html

```diff
@@ -91,14 +91,14 @@
 
         {% else %}
         <div class="download-body">
-            <h1>Downloads for <I><a href="{% url 'work' work.id %}">{{ work.title }}</a></i></h1>
+            <h1 class="h2">Downloads for <I><a href="{% url 'work' work.id %}">{{ work.title }}</a></i></h1>
```

Comment:
I guess making h1 look right is too hard?

Reply:
No, I still think that the h1 stying is fine, but just not for this page where we want to have low size text before the first header. I want to keep the <h1> tag for semantic and a11y reasons

Reply:
I think the h1 styling is always too large on desktop (looks fine on mobile